packages/graphql/src/graphql/ops/FQuestionData.graphql

```diff
@@ -16,5 +16,27 @@ fragment QuestionData on QuestionInstance {
         }
       }
     }
+
+    ... on NumericalQuestionData {
```

Comment:
as the different question types have different schemas for options, we use fragments and spread them based on the parent question type